lib/addons/jupyterhub/index.ts

```diff
@@ -205,13 +205,7 @@ export class JupyterHubAddOn extends HelmAddOn {
             assert(!ingressHosts || ingressHosts.length == 0, 'Ingress Hosts CANNOT be assigned when ingress is disabled');
             assert(!ingressAnnotations, 'Ingress annotations CANNOT be assigned when ingress is disabled');
             assert(!cert, 'Cert option is only supported if ingress is enabled.');
-            setPath(values, "proxy.service", { 
-                "annotations": {
-                    "service.beta.kubernetes.io/aws-load-balancer-type": "nlb",
-                    "service.beta.kubernetes.io/aws-load-balancer-scheme": "internet-facing",
-                    "service.beta.kubernetes.io/aws-load-balancer-nlb-target-type": "ip",
-                }
-            });
+            setPath(values, "proxy.service", {"type": "ClusterIP"});
```

Comment:
hard-coded? Why not let customers decide if they need an NLB and modify our e2e test to set this option to false?